modules/ocf_mirrors/manifests/ftpsync.pp

```diff
@@ -42,11 +43,22 @@
     require => File[$project_path];
   }
 
-  cron { "ftpsync-${title}":
-    command => "BASEDIR=${project_path} ${project_path}/bin/ftpsync > /dev/null 2>&1",
-    user    => 'mirrors',
-    minute  => $cron_minute,
-    hour    => $cron_hour,
-    require => File["${project_path}/bin", "${project_path}/etc/ftpsync.conf"];
+  # TODO: Remove once systemd timers have been tested for reliability
```

Comment:
I'd say you could remove this now if you want given this has been tested for a while.